libs/ethers-provider/src/getDefaultApiKey.ts

```diff
@@ -0,0 +1,36 @@
+import { errors, logger } from './logger'
+
+export const getDefaultApiKey = (network: string) => {
+  switch (network) {
+    case 'optimism-testnet':
+    case 'optimism-mainnet':
+    case 'optimism':
+    case 'arbitrum':
+    case 'arbitrum-mainnet':
+    case 'arbitrum-testnet':
+    case 'homestead':
+    case 'ethereum':
+    case 'mainnet':
+    case 'kovan':
+    case 'ropsten':
+    case 'goerli':
+    case 'rinkeby':
+    case 'optimism-kovan':
+      return '9D13ZE7XSBTJ94N9BNJ2MA33VMAY2YPIRB'
```

Comment:
These are not-my-keys (form the package I sourced this from).

I don't use them, I might remove them (done)